python/phylanx/__init__.py

```diff
@@ -6,3 +6,10 @@
 from .core import *
 from .core.config import *
 from .ast.transducer import Phylanx
+
+
+class parallel:
```

Comment:
Please move this to an appropriate module- perhaps under execution_tree subpackage.

Reply:
Do you have a suggestion?

Reply:
It needs to be in package phylanx.